packages/react/src/Stack/Stack.tsx

```diff
@@ -19,6 +19,9 @@ export const defaultStackSpacing = StackSpacingVariants[1]
 export const StackAlignItemVariants = ['center', 'flex-start', 'flex-end'] as const
 type StackAlignItemVariants = (typeof StackAlignItemVariants)[number]
 
+export const StackFlexWrapVariants = ['wrap', 'nowrap'] as const
```

Comment:
A couple of questions:
- Is the `Flex` prefix really needed? We don't have it on other props that map to flex properties (i.e., `direction`).
- Could this prop be a `boolean` instead of a list of variants? I don't think we plan to support other values such as `wrap-reverse` or `initial`.

Reply:
> Is the Flex prefix really needed? 

Yeah. It was really to avoid this: `wrap="wrap"`, which just felt weird. We also have drawn a line in the sand around this component being a flexbox wrapper, so I felt it was fine to be clear about that in the prop name, but I agree it's not consistent with the pre-existing API. Would you prefer wrap? I don't have super strong opinions on it.

> I don't think we plan to support other values such as wrap-reverse or initial.

I made it a string union just in case we did need to do that in future. It's more future-proofing than anything. I think wrap-reverse could be useful in some scenario's, so maybe worth keeping the API open for those scenario's?

Reply:
If we keep it as a string union, then I think I'd go with flexWrap for clarity. 👍